### DISCUSSION TITLE
Multiple Remote Schemas With Same Schema

### DISCUSSION THREAD

**Discussion by yoshiwarab** [hasura/graphql-engine#7285] [General] [answered]
Hi I'm curious if there's any way to include two remote schemas that have the exact same schema but different data. So for example if I wanted to include two external blogs that exposed the same graphql api, is there any way I could include both of them?

**Comment by motleydev** [comment, marked as answer]
Hi @yoshiwarab, this would be problematic with the nature of a schema. The two ways to go about it would be:

1. Find a way to merge your data sources together (both blogs) and then have the single schema fetch content from essentially the same database.
2. Use a proxy service with something like Apollo to create a namespace for the two apis. Something akin to BlogOneAuthor and BlogTwoAuthor.

Namespacing is making its way into Hasura Cloud in the near future. You can track here for more details. #5863
